receiver/elasticsearchreceiver/metadata.yaml

```
@@ -950,3 +966,28 @@ metrics:
       value_type: int
     attributes: [document_state, index_aggregation_type]
     enabled: false
+  elasticsearch.process.cpu.usage:
+    description: CPU usage in percent, or -1 if not known at the time the stats are computed.
+    unit: '%'
+    gauge:
+      value_type: int
+    attributes: [ ]
+    enabled: false
+  elasticsearch.process.cpu.time:
+    description: CPU time (in milliseconds) used by the process on which the Java virtual machine is running, or -1 if not supported.
+    unit: ms
+    sum:
+      monotonic: true
+      aggregation: cumulative
+      value_type: int
+    attributes: [ ]
+    enabled: false
+  elasticsearch.process.memory:
```

```suggestion
  elasticsearch.process.memory.virtual:
```
See https://github.com/open-telemetry/opentelemetry-specification/blob/f7095ddc1b6cea4e7d42c8462a6def09d19f1326/specification/metrics/semantic_conventions/process-metrics.md?plain=1#L39